### DISCUSSION TITLE
Grouped horizontal bar chart

### DISCUSSION THREAD

**Discussion by binshadpix** [patrykandpatrick/vico#444] [Questions]
How to make grouped horizontal bar chart with this library?

**Comment by Gowsky** [comment]
Hello @binshadpix, Vico currently does not support horizontal bar charts, as mentioned [here](https://github.com/patrykandpatrick/vico/discussions/379#discussioncomment-7323014).

**Reply by binshadpix** [reply]
okay thanks

**Reply by binshadpix** [reply]
and is there any option to limit the points on my line chart on start and end?

**Reply by patrickmichalik** [reply]
Could you elaborate, @binshadpix? What do you mean by “limit the points”?

**Reply by binshadpix** [reply]
![Screenshot 2023-10-23 at 12 12 31 PM](https://github.com/patrykandpatrick/vico/assets/87518309/bec3bd4b-82c9-43a0-aaaa-5c5a46a073d1)

like this points only at start and end of the line

**Reply by patrickmichalik** [reply]
Got it, @binshadpix. I’m afraid this isn’t possible at the moment, but it’d certainly be worth adding such a feature. Could you start a new discussion in the “Ideas” category? Feel free to simply paste a link to your comment in the discussion body.

**Reply by binshadpix** [reply]
okay
